frontends/ast/ast.cc

```diff
@@ -1457,8 +1457,10 @@ RTLIL::IdString AstModule::derive(RTLIL::Design *design, dict<RTLIL::IdString, R
 
 		// Now that the interfaces have been exploded, we can delete the dummy port related to every interface.
 		for(auto &intf : interfaces) {
-			if(mod->wires_.count(intf.first)) {
-				mod->wires_.erase(intf.first);
+			if(mod->wire(intf.first) != nullptr) {
+				pool<RTLIL::Wire*> to_remove;
+				to_remove.insert(mod->wire(intf.first));
```

Comment:
Removing wires is generally an expensive operation. Is it possible to collect all to-be-removed wires and remove them once outside the loop?

Reply:
That is exactly what I thought, and I tried to do that at first.  But somehow that broke the build. :-\

See the Travis logs with and without 60405939943cd812e146b84848be8bc9307702db

Reply:
Sorry, I didn't notice you had tried that already. I figured out what was going on, let me add it as a comment.